src/components/courses/CourseSearch.js

```diff
@@ -22,6 +22,7 @@ const CourseSearch = () => {
     };
 
     getCourses(params);
+    getColumnOptions();
```

Comment:
This should only happen once per session. Should put this in a separate `useEffect`

```js
useEffect(() => {
  getColumnOptions();
}, []);
```